### ISSUE TITLE
there are 2 small mistake in retina_unet and detection_unet

### ISSUE THREAD

**Issue by CodingHHW** [MIC-DKFZ/medicaldetectiontoolkit#91]
in models/retina_unet line 378, should change norm=False to norm=self.cf.norm
in models/detection_unet line 134, should change 'monitor_extra_values' to 'monitor_values'

**Comment by Gregor1337** [comment]
Hi CodingHHW,
thank you for your input.
The issues are fixed in [this commit](https://github.com/MIC-DKFZ/medicaldetectiontoolkit/commit/a58d397f53a9d1ae425cb75a6869da603650da5e) and [this commit](https://github.com/MIC-DKFZ/medicaldetectiontoolkit/commit/3ef7d8974aa0de274fe5dcbbc23975ad30e5da01).

Regarding "models/retina_unet line 378, should change norm=False to norm=self.cf.norm": norm=None is what was meant here since it is the final convolution producing logits that are normalized by a softmax and hence don't require further normalization.
